WordPressKitTests/DomainsServiceRemoteRESTTests.swift

```diff
@@ -114,4 +115,25 @@ class DomainsServiceRemoteRESTTests: RemoteTestCase, RESTTestable {
         
         waitForExpectations(timeout: timeout, handler: nil)
     }
+    
+    func testGetStatesSuccess() {
```

Comment:
I think it would be also useful to add a test that verify that we handle the countries that have no meaningful division into "states" correctly. (like `DE` or `PL` for example)

Reply:
for those countries the service returns empty array so I added a new test case for handling empty result, is that what you meant?